(INFO)[utils.py:341]  $AICROWD_DATASET_NAME=celebA
(INFO)[utils.py:342]  $DATASET_NAME=celebA
(INFO)[utils.py:343]  $DISENTANGLEMENT_LIB_DATA=/home/emanuele.marconato/data
(INFO)[base_disentangler.py:24]  Device: cuda
(INFO)[data_loader.py:635]  Datasets root: /home/emanuele.marconato/data
(INFO)[data_loader.py:636]  Dataset: celebA
(INFO)[data_loader.py:317]  include_labels: None
(INFO)[base_disentangler.py:87]  Number of samples: 202599
(INFO)[base_disentangler.py:88]  Number of batches per epoch: 1621
(INFO)[base_disentangler.py:89]  Number of channels: 3
0it [00:00, ?it/s][33;40m(WARNING)[base_disentangler.py:443]  Could not load G[0m
[33;40m(WARNING)[base_disentangler.py:444]  Error(s) in loading state_dict for VAEModel:
	size mismatch for encoder.main.13.weight: copying a param with shape torch.Size([64, 1024]) from checkpoint, the shape in current model is torch.Size([128, 1024]).
	size mismatch for encoder.main.13.bias: copying a param with shape torch.Size([64]) from checkpoint, the shape in current model is torch.Size([128]).
	size mismatch for decoder.main.1.weight: copying a param with shape torch.Size([256, 32, 1, 1]) from checkpoint, the shape in current model is torch.Size([256, 64, 1, 1]).[0m
0it [00:02, ?it/s](INFO)[base_disentangler.py:466]  Model Loaded: checkpoints/celebA-Padless/CBM_join_sup_100/last @ iter:0, lr:0.000001
Traceback (most recent call last):
  File "main.py", line 77, in <module>
    main(_args)
  File "main.py", line 60, in main
    model.test(out_path=out_path)
  File "/home/emanuele.marconato/disentanglement-pytorch/models/cbm_join.py", line 365, in test
    z_array[self.batch_size * internal_iter:self.batch_size * internal_iter + self.batch_size, :] = z
ValueError: could not broadcast input array from shape (60,64) into shape (100,64)

